packages/web-components/src/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,95 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/display.js';
+import {
+  borderRadiusMedium,
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  colorNeutralShadowAmbient,
+  colorNeutralShadowKey,
+  fontFamilyBase,
+  fontSizeBase200,
+  lineHeightBase200,
+  spacingHorizontalXS,
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+import { TooltipPositioning } from './tooltip.options.js';
+
+/**
+ * Styles for the tooltip component
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host(:not(:popover-open)) {
+    display: none;
+  }
+
+  :host {
+    --blockOffset: ${spacingVerticalXS};
+    --inlineOffset: ${spacingHorizontalXS};
+    background: ${colorNeutralBackground1};
+    border-radius: ${borderRadiusMedium};
+    border: 1px solid var(--colorTransparentStroke);
+    box-sizing: border-box;
+    color: ${colorNeutralForeground1};
+    display: inline-flex;
+    filter: drop-shadow(0 0 2px ${colorNeutralShadowAmbient}) drop-shadow(0 4px 8px ${colorNeutralShadowKey});
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase200};
+    inset-area: block-start;
+    line-height: ${lineHeightBase200};
+    margin: 0;
```

Comment:
Does this need to be set?

Reply:
Good question, yes it does. `[popover]` applies a `margin: auto` in the default stylesheet. So we need this to `unset` that value otherwise it tries to center itself. Three options:

1. Keep as `margin: 0`
2. Update to `margin: unset` for clarity
3. Create two declarations to explicitly `unset` in the `:host(:is([positioning^...` rules.

Reply:
Updated to `margin: unset`